_posts/2021-10-04-sharing-materials-from-ncwit-talk.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Sharing resources on counteracting bias from recent NCWIT talk
```

Comment:
```suggestion
title: Sharing resources on counteracting bias, from recent NCWIT talk
```

Reply:
good catch, we don't it to sound like we are counteracting bias from the talk 😬 

Re-ordered the title: "Sharing resources on counteracting bias from recent NCWIT talk"